src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,98 @@
+import { useMemo } from 'react';
+import classnames from 'classnames';
+import { Status } from '../../types/Status';
+import { useFilter } from '../../context/FilterContext';
+import { useTodo } from '../../context/TodoContext';
+import { deleteTodo } from '../../api/todos';
+import { useError } from '../../context/ErrorContext';
+
+export const TodoFooter = () => {
+  const { selectedFilter, setSelectedFilter } = useFilter();
+  const { todos, setTodos } = useTodo();
+  const { setErrorMessage } = useError();
+
+  const isActiveItemsLeft = todos
+    .filter(({ completed }) => !completed).length;
+
+  const hasCompletedTodo = useMemo(() => {
+    return todos.some(todo => todo.completed);
+  }, [todos]);
+
+  const clearCompleted = () => {
+    const deletePromises = todos
+      .filter((todo) => todo.completed)
+      .map(({ id }) => deleteTodo(id));
+
+    Promise.all(deletePromises)
+      .then(() => {
+        setTodos((prevState) => prevState.filter((todo) => !todo.completed));
+      })
+      .catch(() => {
+        setErrorMessage('Unable to delete a todo');
+      });
+  };
+
+  return (
+    <footer data-cy="Footer" className="todoapp__footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {isActiveItemsLeft > 1
+          ? `${isActiveItemsLeft} items left`
+          : `${isActiveItemsLeft} item left`}
```

Comment:
You only have to check one word:
```suggestion
        {`${isActiveItemsLeft} ${isActiveItemsLeft > 1 ? 'items' : 'item'}  left`}
```